xml_feed/xml_feed.py

```diff
@@ -49,6 +51,7 @@ def _parse_articles(cls, raw_feed):
     @staticmethod
     def _parse_articles_rss(root):
         used_titles = set()
+        content_key = '{http://purl.org/rss/1.0/modules/content/}encoded'
```

Comment:
Can you make this a global array? Then change the test condition to be `if child.tag in CONTENT_KEYS:`

Reply:
Hi @Zoramite I did a refactor to use an array called CONTENT_KEYS.

I tried using a dict to simplify further but I couldn't find a way to make the string '{http://purl.org/rss/1.0/modules/content/}encoded' behave correctly within a dict.

This is roughly the idea that I couldn't get to work:
article[child.tag] = child.text.encode('utf8')

All variations I tried broke when child.tag is '{http://purl.org/rss/1.0/modules/content/}encoded'

and article['child.tag'] doesn't work because the article instance doesn't allow assignment.

Ideally, there would be a mapping like:
'title': 'title',
'description': 'description',
'published': 'pubDate',
'content': '{http://purl.org/rss/1.0/modules/content/}encoded'

Let me know if this is ok or if you have any ideas to make it better.